sdk/src/manifest.rs

```diff
@@ -1033,8 +1033,8 @@ impl Manifest {
         // todo:: see if we can pass a trait with to_vec support like we to for Strings
         let asset = asset.to_vec();
         let mut stream = std::io::Cursor::new(asset);
-        self.embed_stream(format, &mut stream, signer)?;
-        Ok(stream.into_inner())
+
+        self.embed_stream(format, &mut stream, signer)
```

Comment:
When I first wrote this it always replaced the input stream, but didn't update this function when when the logic changed to generate a new output stream. A good reason why every function, no matter how thin a wrapper, should have a unit test!